content/examples/docker/basic.docker-compose.yml.md

```diff
@@ -1,25 +1,16 @@
 ```yaml
 version: "3"
-networks:
-  main: {}
 services:
   pomerium:
-    image: pomerium/pomerium:latest
+    image: pomerium/pomerium:v0.22.0
```

Comment:
We should update this to the latest patch _everywhere_. And we need to remember to bump this everytime we do a release or minor.

Reply:
let's just do the `latest => v0.22.0` _everywhere_ part in a separate PR, as I have an incoming PR this one is blocking

Reply:
Let's do this in a FUP PR. But let's change the format to use minor'tagged images e.g. (`v0.21`)